### PR TITLE
clean old onboarding and communities

### PR DESCRIPTION
**Pull request by flexsurfer**
clean old onboarding and communities

### CODE REVIEW

**Review by ilmotta — APPROVED** [review]
Gotta love these PRs, thanks @flexsurfer

File: src/status_im/ui/screens/communities/create.cljs [hunk 1]
```diff
@@ -151,7 +150,7 @@
 
 (defn form
   []
-  (let [{:keys [name description membership editing?]} (rf/sub [:communities/create])]
+  (let [{:keys [name description editing?]} (rf/sub [:communities/create])]
```
Comment (J-Son89) [inline src/status_im/ui/screens/communities/create.cljs:153]: just to check, it's still possible to get the old ui to create a community right? I believe we don't have that functionality in new communities yet. But perhaps this is actually a good thing as the mobile does not add the banner photo.

**Review by J-Son89 — APPROVED** [review]

File: src/status_im/ui/screens/communities/create.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/communities/create.cljs:153]: its possible but only for e2e purposes, by design it should be possible to create a community only on desktop

Comment (J-Son89) [reply src/status_im/ui/screens/communities/create.cljs:153]: Ok cool, good to know! 👌

**Comment by pavloburykh** [comment]
@flexsurfer thanx for the PR. Performed basic check of onboarding and communities. No issues form my side. e2e failures are not PR related. Ready for merge.

**Pull request merged**